Project/GOMS-iOS-Admin/Sources/Presentation/Scene/Main/Outing/Reactor/OutingReactor.swift

```diff
@@ -37,7 +37,7 @@ class OutingReactor: Reactor, Stepper {
         case fetchStudentList(studentList: [StudentListResponse])
     }
     
-    struct OutingItem {
+    public struct OutingItem {
```

Comment:
```suggestion
   struct OutingItem {
```
internal로 바꾸는 건 어떤가요?